### ISSUE TITLE
How can we fix the problems we find during code review?

### ISSUE THREAD

**Issue by JunyangPan** [CISC-CMPE-327/Information#68]
How can we fix the problem we find during code review? Fix it in location and push to github and create another pull request?

**Comment by steven-hh-ding** [comment]
> Do we need to merge the code to master for A2 or we just do the pull request?
Yes, to create a tag (of everything you have at the moment) you need to merge. 

> number of pull-request reviews done by xxx
Number of pull-requests one have checked (making comments and approving). If each of you only made 1 PR yes it is max 3. 

> fixing problem
The guy who created the pull-request can continue pushing to the branch and you will see the updated code in the original PR.

> test plan
As long as you cover the required items in the list using to the best of what you know that should be fine
